<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(208, 253, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>741</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(60, 182, 182);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Качественные реакции в органике&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>121</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <widget class="QRadioButton" name="alkanes">
       <property name="text">
        <string>Алкан</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="alkenes">
       <property name="text">
        <string>Алкен</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="alkynes">
       <property name="text">
        <string>Алкин</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="cycloalkanes">
       <property name="text">
        <string>Циклоалкан</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="alkadienes">
       <property name="text">
        <string>Алкадиен</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="alcohols">
       <property name="text">
        <string>Спирт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="aldehyde">
       <property name="text">
        <string>Альдегид</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="arenas">
       <property name="text">
        <string>Бензол</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="phenol">
       <property name="text">
        <string>Фенол</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="carboxylic">
       <property name="text">
        <string>Карбоновая кислота</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="amines">
       <property name="text">
        <string>Амин</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>171</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="permangatan">
       <property name="text">
        <string>Раствор перманганата калия</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="iodine">
       <property name="text">
        <string>Раствор йода</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="burn">
       <property name="text">
        <string>Горение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="silver">
       <property name="text">
        <string>Аммиачный раствор серебра</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="copper">
       <property name="text">
        <string>Соли меди</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="copper_oxide">
       <property name="text">
        <string>Оксид меди при температуре</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="copper_hydroxide">
       <property name="text">
        <string>Свежеосажденный Cu(OH)2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="nitrating_mixture">
       <property name="text">
        <string>Нитрующая смесь</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ferric_chloride">
       <property name="text">
        <string>Раствор хлорида железа (III)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="metals">
       <property name="text">
        <string>Активные металлы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="litmus">
       <property name="text">
        <string>Лакмус</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="bromine_water">
       <property name="text">
        <string>Бромная вода</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="plus">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>plus.png</normaloff>plus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>474</x>
      <y>200</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_result">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>56</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>equals.png</normaloff>equals.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_return">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>380</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(178, 255, 234);</string>
    </property>
    <property name="text">
     <string>в начало</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
